link/src/main/java/com/meshconnect/link/ui/LinkActivity.kt

```diff
@@ -202,7 +226,15 @@ internal class LinkActivity : AppCompatActivity() {
 
         override fun onPageFinished(view: WebView?, url: String?) {
```

Comment:
iOS SDK calls `evaluateJavascript` when `loaded` event is received, as I remember it may not be exactly the moment when `onPageFinished` function is called

Reply:
Good point